app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,28 @@
+class TodosController < ApplicationController
+  protect_from_forgery with: :null_session
+  def index
```

Comment:
Nice and simple here on these first two. You could make them even more straight forward if you wanted with:

```ruby
def index
  render json: Todo.all
end
end

def show
  render json: Todo.find(params[:id])
end
```

These are short enough lines that it doesn't effect readability to me and it will remove setting a variable where it may not be needed.

I think later you can expand these methods with better error handling, pagination, etc, but this would be a bit cleaner in the interim for me.